src/book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  def initialize(title, author)
+    @title = title,
```

Comment:
- [ ] I don't think this comma is necessary and would throw errors. Kindly remove it.